lib/jekyll-admin/server/page.rb

```diff
@@ -34,6 +39,18 @@ class Server < Sinatra::Base
 
       private
 
+      def html_content?
```

Comment:
Why can't we just [use `html?` here?](https://github.com/jekyll/jekyll/blob/master/lib/jekyll/page.rb#L167-L170) (like we use for `pages` below)? If the file doesn't exist yet, we can simply create the page object, rather than reimplementing potentially divergent logic.